python/ray/train/constants.py

```diff
@@ -16,21 +16,20 @@
 )
 
 # Autofilled session.report() metrics. Keys should be consistent with Tune.
-TIMESTAMP = "_timestamp"
-TIME_THIS_ITER_S = "_time_this_iter_s"
-TRAINING_ITERATION = "_training_iteration"
+# The train provided `TIME_THIS_ITER_S` and `TIMESTAMP` will triumph what's
+# auto-filled by Tune session.
+TIMESTAMP = "timestamp"
+TIME_THIS_ITER_S = "time_this_iter_s"
```

Comment:
If they need to be consistent, can we just import them from tune?

Reply:
yeah makes sense, but can we do that after this? https://github.com/ray-project/ray/issues/32260

Reply:
Also added a TODO